compiler/src/iree/compiler/Dialect/HAL/Target/ROCM/ROCMTarget.cpp

```diff
@@ -199,9 +201,16 @@ class ROCMTargetBackend final : public TargetBackend {
     iree_hal_rocm_ExecutableDef_start_as_root(builder);
 
     // Link module to Device Library
+    std::string rocmBitcodeDir = clROCMBitcodeDir;
     if (clROCMLinkBC) {
-      linkROCDLIfNecessary(llvmModule.get(), clROCMTargetChip,
-                           clROCMBitcodeDir);
+      if (clROCMBitcodeDir.empty()) {
+        return variantOp.emitError()
+               << "cannot find ROCM bitcode files. Check your installation "
+                  "consistency and in the worst case, set --iree-rocm-bc-dir= "
+                  "to "
+                  "an explicit location on your system.";
```

Comment:
nit: re-flow comment

```suggestion
        return variantOp.emitError()
               << "cannot find ROCM bitcode files. Check your installation "
                  "consistency and in the worst case, set --iree-rocm-bc-dir= "
                  "to an explicit location on your system.";
```